modules/reference/pages/systems/system-details/sd-virtualization.adoc

```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
 image::system_details_traditional_virtualization_deployment.png[scaledwidth=80%]
 
 When the deploment scheduled it is listed as an action on the menu:Main Menu[Schedule > Pending Actions].
+
+== Storage [Salt]
+
+The [guimenu]``Storage`` tab presents a tree list of the virtual storage pools and volumes defined on the virtual host.
+The first level of the tree is the list of storage pools and all items contained in them are volumes.
+By default the pools need to be expanded to show the volumes.
+
+image::modules/reference/assets/images/system_details_salt_virtual_pools.png[scaledwidth=80%]
+
+For each pool the following information is provided:
+
+[guimenu]``Status``::
+This field indicates whether the pool is running or stopped.
+
+[guimenu]``Autostart``::
+This field indicates whether the pool is started automatically when the virtual host boots
+
+[guimenu]``Persistent``::
+This field indicates whether the pool will be kept or not after being stopped.
+
+[guimenu]``Location``::
+This field indicates the target path of the storage pool.
+Note that some pool types don't have any associated path.
+
+[guimenu]``Usage``::
+This field shows the disk usage of the pool or [guimenu]``Unknown`` if the pool is not running.
+
+For each volume the following information is provided:
+
+* The names of the virtual machines using the volume.
+This list may not be provided depending on the pool type.
+
+* The disk usage of the volume.
+
+=== Refreshing a pool
+
+[systemitem]``Libvirt`` doesn't update the pool usage and contents in real time.
+In order to update the usage or to see a volume that has been created outside of {productname}, the pool needs to be refreshed.
+Click the [guimenu]``Refresh`` button to schedule a refresh of the pool.
+
+=== Creating a pool
+
+To create a storage pool click the [guimenu]``Create Pool`` button.
```

Comment:
This should be a procedure.

Reply:
OK, how is that supposed to appear? prefixed with `[procedure]`?

Reply:
I've edited, please confirm.

Reply:
it's good for me